test/reporters/base.spec.js

```diff
@@ -312,6 +312,7 @@ describe('Base reporter', function() {
   });
 
   it('should interpret Chai custom error messages', function() {
+    this.timeout(1000);
     var chaiExpect = require('chai').expect;
```

Comment:
Instead increasing timeout, how about move `require('chai')` to outside `it`. requiring chai takes most of time.

```
var chaiExpect = require('chai').expect;
it('should interpret Chai custom error messages', function() {
  try {
    chaiExpect(43, 'custom error message').to.equal(42);
  } catch (err) {
...
```

In my local, original test time took `46ms` but moving `require('chai')` took only `8ms`.

Reply:
Someone went to the effort to minimize the scope to _just_ this test.
This was considered, but didn't wish to increase its scope.

I have no idea about possible interactions between Chai and Unexpected, and have no desire to go down that rabbit hole. But you can borrow my flashlight...

Reply:
What interactions between `chai` and `unexpected`?